pkg/bucket/lifecycle/lifecycle.go

```diff
@@ -275,3 +326,13 @@ func (lc Lifecycle) PredictExpiryTime(obj ObjectOpts) (string, time.Time) {
 	}
 	return finalExpiryRuleID, finalExpiryDate
 }
+
+// GetTransitionARN returns transition ARN.
+func (lc Lifecycle) GetTransitionARN(obj ObjectOpts) string {
```

Comment:
Can we call this GetTransitionStorageClass ? since we are in lifecycle package, it will be less confusing that way

Reply:
done